ui/src/app/views/homepage/component.spec.ts

```diff
@@ -100,31 +109,38 @@ describe('HomepageComponent', () => {
   it('should display quick tour when clicked', fakeAsync(() =>  {
     loadProfileWithPageVisits({pageVisits: [{page: 'homepage'}]});
     updateAndTick(fixture);
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithClass(
+        renderIntoDocument(React.createElement(
```

Comment:
Sorry if I missed it, but is there background around why this element would need to be manually rendered like this? Shouldn't that just naturally be triggered by Angular?

Reply:
The jasmine tests do some weird abstractions and optimizations. I think we ended up figuring out that it blocked the procedural generation like that? Not entirely sure. I can poke around some to figure out the why.

Reply:
Doesn't necessarily need to block this PR, but I'm not sure how effective the tests will be if we need to manually render React elements before the parent component can be tested. This might be changing anyways if we're looking at Jest, though.